src/commands.js

```diff
@@ -474,8 +474,20 @@ export function goToNextCell(direction) {
     let cell = findNextCell(selectionCell(state), direction)
     if (cell == null) return
     if (dispatch) {
-      let $cell = state.doc.resolve(cell)
-      dispatch(state.tr.setSelection(TextSelection.between($cell, moveCellForward($cell))).scrollIntoView())
+      if (dispatch) {
+        const cellNode = state.doc.nodeAt(cell)
+        const cellLastChild = cellNode.lastChild
+        if (cellLastChild == null) return
```

Comment:
I guess we should return false here but then this is JavaScript not TypeScript and I see the return statement above is also like this so 🤷

Reply:
Yeah, I don't know what conventions to follow in this repo. I went ahead and added return false to both of these.